Fix for additional separators appearing when scripts are disabled

pyrophone opened the pull request:
Fix for issue #27866. Moved a separator to a different place.

![gdScreen1](https://user-images.githubusercontent.com/15381918/55974622-9a3c0880-5c77-11e9-87e3-fe9b9d648502.png)

![gdScreen2](https://user-images.githubusercontent.com/15381918/55974666-b9d33100-5c77-11e9-8d1b-4a92855bd54b.png)

![godot3](https://user-images.githubusercontent.com/15381918/56050282-fd42a380-5d3a-11e9-93cb-698cb7017a90.png)

![godot4](https://user-images.githubusercontent.com/15381918/56050289-02075780-5d3b-11e9-9cf6-1dd0aef78ae6.png)

pyrophone committed:
Fixed scene tree bug when scripts are disabled

Xrayez commented:
Yeah I think this should fix the issue, some other features could be disabled/enabled to ensure no extra separators appear.

I couldn't test your changes because your branch has build bugs on MinGW buildsystem inherited from master branch. Consider rebasing latest fixes from master:

http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase

```
git add .
git commit --amend
git push --force origin SceneTreeScriptFix
```

Also, it's better to link the issue number directly in the message body, so that it will be linked to the issue properly, and because of that you'll also need to come up with a better name for this pull request.

pyrophone commented:
I've rebased and updated the commit, as well as the PR name. I've also attached a screenshot of having scene tree dock editing disabled, and a screenshot of having scripts enabled just in case.

Xrayez commented:
Yeah it works just fine. Lets wait for people with merge access. 😛

akien-mga commented:
Thanks!
